pkg/bpf/ads/trace_point.go

```diff
@@ -34,10 +42,24 @@ type BpfTracePoint struct {
 	bpf2go.KmeshTracePointObjects
 }
 
-func (sc *BpfTracePoint) NewBpf(cfg *options.BpfConfig) {
-	sc.Info.MapPath = cfg.BpfFsPath
-	sc.Info.BpfFsPath = cfg.BpfFsPath
+func (sc *BpfTracePoint) NewBpf(cfg *options.BpfConfig) error {
+	sc.Info.MapPath = cfg.BpfFsPath + "/bpf_kmesh/map/"
```

Comment:
Can you list the dirs/files in /sys/fs/bpf? I think these directory structures need to be reviewed to avoid unreasonable directory division.

Reply:
[root@kmesh-z2szb bpf_kmesh]# tree
.
|-- map
|   |-- kmesh_cluster
|   |-- kmesh_cluster_stats
|   |-- kmesh_config_map
|   |-- kmesh_events
|   |-- kmesh_listener
|   |-- kmesh_manage
|   |-- kmesh_map1600
|   |-- kmesh_map192
|   |-- kmesh_map296
|   |-- kmesh_map64
|   |-- kmesh_version
|   |-- map_of_cluster_eps
|   |-- map_of_cluster_eps_data
|   |-- map_of_cluster_sock
|   |-- map_of_router_config
|   |-- map_of_sock_storage
|   |-- outer_of_maglev
|   |-- tmp_buf
|   `-- tmp_log_buf
|-- sockconn
|   |-- prog_link
|   `-- tail_call_map
|-- sockops
|   |-- prog_link
|   `-- tail_call_map
`-- tracepoint
    `-- prog_link

5 directories, 24 files

Reply:
/sys/fs/bpf